SKILL.tsv

```diff
@@ -4796,10 +4796,10 @@ SKILL_20150717_004795	$Flesh Cannon: Decay
 SKILL_20150717_004796	$Enemies hit by [Flesh Cannon] have a chance to be afflicted with Decay for 15 seconds. Increases the chance by 10% per attribute level.
 SKILL_20150717_004797	Flesh Hoop: Demoralize
 SKILL_20150717_004798	Attak of enemies nearby the hoop after using [Flesh Hoop] decrease for 4 seconds. [Weakened ATK] can be overlapped max 10 times.
-SKILL_20150717_004799	The enemies near Shoggoth will fall into Decay status for 15 secs. This ability will last for 1 min per Attribute level after Shoggoth gets summoned.
+SKILL_20150717_004799	The enemies near Shoggoth will fall into Decay status for 15 secs. This ability will last for 1 min per attribute level after Shoggoth gets summoned.
 SKILL_20150717_004800	$Increases the range of [Disinter] by 10 per attribute level.
 SKILL_20150717_004801	$Decreases the cooldown of [Dig] by 10 seconds.
-SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 mins for the allies that participated in [Item Awakening] dungeon. ATK will increase by 20% per Attribute level.
+SKILL_20150717_004802	Physical ATK and Magic ATK will be increased for 30 minutes for the allies that participated in the [Item Awakening] dungeon. ATK will increase by 20% per attribute level.
```

Comment:
Increases the physical attack and magic attack for 30 minutes of allies that have participated in a [Item Awakening] dungeon. Increases the attack by 20% per attribute level.